src/app/components/pages/TeachingOrders.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement, useEffect, useState } from "react";
+import { Col, Container, Row } from "reactstrap";
+import { Tabs } from "../elements/Tabs";
+import { PageFragment } from "../elements/PageFragment";
+import { TitleAndBreadcrumb } from "../elements/TitleAndBreadcrumb";
+import { MetaDescription } from "../elements/MetaDescription";
+
+enum TEACHING_ORDER_STAGE {
+    CORE_Y1 = "Core Y1",
+    CORE_Y2 = "Core Y2",
+    ADVANCED_Y1 = "Advanced Y1",
+    ADVANCED_Y2 = "Advanced Y2"
+}
+
+export const TeachingOrders = () => {
+    const TEACHING_ORDER_STAGES = Object.keys(TEACHING_ORDER_STAGE);
+    const [stageTab, setStageTab] = useState<typeof TEACHING_ORDER_STAGES[number]>(TEACHING_ORDER_STAGES[0]);
+    const [stageTabOverride, _setStageTabOverride] = useState<number | undefined>(TEACHING_ORDER_STAGES.indexOf(stageTab) + 1 || undefined);
```

Comment:
This doesn't currently have any functionality to read the hash anchor. Something like this should work:

```suggestion

const sanitiseStageTab = (stageTab: string) => stageTab.replace(" ", "_").toLowerCase();
const sanitisedStageTabsMap = Object.fromEntries(Object.keys(TEACHING_ORDER_STAGE).map(stage => [sanitiseStageTab(stage), stage]));

const getStageFromURL = () : TEACHING_ORDER_STAGE | undefined => {
    const urlStage = window.location.hash.slice(1);
    return sanitisedStageTabsMap[urlStage] as TEACHING_ORDER_STAGE | undefined;
};

export const TeachingOrders = () => {
    const TEACHING_ORDER_STAGES = Object.keys(TEACHING_ORDER_STAGE);
    const [stageTab, setStageTab] = useState<typeof TEACHING_ORDER_STAGES[number]>(getStageFromURL() || TEACHING_ORDER_STAGE.CORE_Y1);
    const [stageTabOverride, _setStageTabOverride] = useState<number | undefined>(Object.values(sanitisedStageTabsMap).indexOf(stageTab) + 1 || undefined);
```

Reply:
(Admittedly, this won't read changes made to the hash anchor without reloading, but we don't do that anywhere. Something for another time maybe..)